charts/dashmachine/README.md

```diff
@@ -0,0 +1,33 @@
+dashmachine
+===========
+DashMachine is another web application bookmark dashboard, with fun features.
+
+Current chart version is `1.0.0`
```

Comment:
You can leave the current chart version out

Reply:
Removed